packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+// import {Item, Picker, SearchField, SearchWithin} from '../';
+// import {Provider} from '@react-spectrum/provider';
+// import React from 'react';
+// import {render} from '@testing-library/react';
+// import scaleMedium from '@adobe/spectrum-css-temp/vars/spectrum-medium-unique.css';
+// import themeLight from '@adobe/spectrum-css-temp/vars/spectrum-light-unique.css';
+
+// let theme = {
+//   light: themeLight,
+//   medium: scaleMedium
+// };
+
+describe('SearchWithin', function () {
+
+  // let defaultProps = {
+  //   label: 'Test'
+  // };
+
+  // const ExampleSearchWithin = React.forwardRef((props = {}, ref) => (
+  //   <Provider theme={theme}>
+  //     <SearchWithin {...defaultProps} {...props} ref={ref}>
+  //       <SearchField placeholder="Search" />
+  //       <Picker defaultSelectedKey="all">
+  //         <Item key="all">All</Item>
+  //         <Item key="campaigns">Campaigns</Item>
+  //         <Item key="audiences">Audiences</Item>
+  //         <Item key="tags">Tags</Item>
+  //       </Picker>
+  //     </SearchWithin>
+  //   </Provider>
+  //   ));
+
+  // function renderSearchWithin(props = {}) {
+  //   return render(<ExampleSearchWithin {...props} />);
+  // }
+  
+  it('temp test', function () {
```

Comment:
good tests to add
```
can type in search and get onChange
can open menu and get onChange
```
while both of those test SearchField and Picker respectively, it makes sure we don't accidentally replace those props

```
searchfield and picker are labelled correctly
```